templates/bundle/resources/usr/local/bin/liferay_entrypoint.sh

```diff
@@ -83,4 +85,19 @@ function start_monitor_liferay_lifecycle {
 	fi
 }
 
+function start_interval_thread_dump {
+	while [ ! -z "${LIFERAY_DOCKER_AUTO_THREAD_DUMP_SLEEP_FILE}" ]
+	do
+		generate_thread_dumps_now
+	done &
+}
+
+function generate_thread_dumps_now {
+	while [ -s "${LIFERAY_DOCKER_AUTO_THREAD_DUMP_SLEEP_FILE}" ]
+	do
+		/usr/local/bin/generate_thread_dump.sh
```

Comment:
generate_thread_dump will generate 20 thread dumps by default, right? 

Also, you need to read how long it should sleep from the LIFERAY_DOCKER_AUTO_THREAD_DUMP_SLEEP_FILE file.

Reply:
This will require some changes in the logic: 
Until we have the LIFERAY_DOCKER_AUTO_THREAD_DUMP_SLEEP_FILE, do thread dump, sleep for the amount of time which is defined in the LIFERAY_DOCKER_AUTO_THREAD_DUMP_SLEEP_FILE, then if the file disappears, check every minute.